bazel/grpc_deps.bzl

```diff
@@ -382,6 +377,16 @@ def grpc_test_only_deps():
         actual = "@com_github_yaml_pyyaml//:yaml",
     )
 
+    native.bind(
+        name = "absl-flags",
```

Comment:
In the root, they appear to use `absl/flags:flag`, so if you'd prefer, I could see if I could do that with or without the `native.bind`. I have to admit the `external_deps` confused me a bit.

Reply:
Removed `native.bind` and now these just use `absl/flags:flag` like most of the other abseil external dependencies.

Reply:
I like your approach and I was thinking of removing those special binding. These were created before having a full abseil to address their specific needs but it's better to consolidate them. Thanks for the clean-up!